types/common.ts

```diff
@@ -1,5 +1,14 @@
 import axiosStatic, { AxiosInstance } from "axios"
 
+export interface UnimplementedFields {
+    /**
+     * Support arbitrary keys (to make this type useful even when it has drifted from the real implementation)
+     */
+    [k: string]: unknown
+}
+
```

Comment:
I really like the new type for the unimplemented relationships.
Let's add the Relationship[] and undefined as an option, and move it to the common file so it can be used by other types.
```suggestion
export interface UnimplementedRelationships {
    /**
     * Support arbitrary keys (to make this type useful even when it has drifted from the real implementation)
     */
    [k: string]: Relationship | Relationship[] | undefined
}
```